src/tests/messageUtils.ts

```diff
@@ -1,50 +1,62 @@
 /**
- * Copyright (c) 2018-2024, Built on KILT.
+ * Copyright (c) 2025, Built on KILT.
  *
  * This source code is licensed under the BSD 4-Clause "Original" license
  * found in the LICENSE file in the root directory of this source tree.
  */
 
-import {
-  KiltKeyringPair,
+import { NewLightDidVerificationKey, didKeyToVerificationMethod, getFullDid, getStoreTx } from '@kiltprotocol/did'
+import type {
+  Did,
+  DidDocument,
+  DidUrl,
   KeyringPair,
   KiltEncryptionKeypair,
-  DidDocument,
-  NewLightDidVerificationKey,
-  KeyRelationship,
-  DidServiceEndpoint,
-  DidKey,
-  DidVerificationKey,
-  DecryptCallback,
-  EncryptCallback,
+  KiltKeyringPair,
+  Service,
+  SignerInterface,
+  VerificationMethod,
+  VerificationRelationship,
 } from '@kiltprotocol/types'
-
-import { Did, SignCallback } from '@kiltprotocol/sdk-js'
-import { Crypto } from '@kiltprotocol/utils'
-import { blake2AsU8a, blake2AsHex } from '@polkadot/util-crypto'
+import { Crypto, Signers } from '@kiltprotocol/utils'
+import { blake2AsHex, blake2AsU8a } from '@polkadot/util-crypto'
+import type { DecryptCallback, EncryptCallback } from '../types/Message.js'
 
 /**
  * Generates a callback that can be used for signing.
  *
  * @param keypair The keypair to use for signing.
  * @returns The callback.
  */
-export function makeSignCallback(keypair: KeyringPair): KeyToolSignCallback {
-  return (didDocument) =>
-    async function sign({ data, keyRelationship }) {
-      const keyId = didDocument[keyRelationship]?.[0].id
-      const keyType = didDocument[keyRelationship]?.[0].type
-      if (keyId === undefined || keyType === undefined) {
-        throw new Error(`Key for purpose "${keyRelationship}" not found in did "${didDocument.uri}"`)
+export function makeDidSigners(
```

Comment:
This helper doesn't seem to be used everywhere. Maybe I misunderstood what this helper is doing.